seatunnel-apis/seatunnel-api-flink/src/main/java/org/apache/seatunnel/flink/FlinkEnvironment.java

```diff
@@ -322,4 +326,61 @@ private void setCheckpoint() {
         }
     }
 
+    public StreamExecutionEnvironment creatMetricStreamEEnvironment() {
+
+        if(!config.hasPath(ConfigKeyName.Metric_Class)){
+            return StreamExecutionEnvironment.getExecutionEnvironment();
+        }
+        //构建flink-metrics参数
+        ConfigOption<String> REPORTERS_LIST =
+                key("metrics.reporters")
+                        .stringType()
+                        .noDefaultValue();
+
+        ConfigOption<String> REPORTER_CLASS =
+                key("metrics.reporter.seatunnel_reporter.class")
+                        .stringType()
+                        .noDefaultValue();
+        ConfigOption<Duration> REPORTER_INTERVAL =
+                key("metrics.reporter.seatunnel_reporter.interval")
+                        .durationType()
+                        .defaultValue(Duration.ofSeconds(10));
+
+        ConfigOption<String> REPORTER_CONFIG_PORT =
+                key("metrics.reporter.seatunnel_reporter.port")
+                        .stringType()
+                        .noDefaultValue();
+        ConfigOption<String> REPORTER_CONFIG_HOST =
+                key("metrics.reporter.seatunnel_reporter.host")
+                        .stringType()
+                        .noDefaultValue();
+        ConfigOption<String> REPORTER_CONFIG_JOB_NAME =
+                key("metrics.reporter.seatunnel_reporter.jobName")
+                        .stringType()
+                        .noDefaultValue();
+
+        Configuration seatunnel_reporter = new Configuration().set(REPORTERS_LIST, "seatunnel_reporter").set(REPORTER_CLASS, "org.apache.seatunnel.metrics.flink.SeatunnelMetricReporter");
```

Comment:
use `lower camel case` for field and parameter. Change `seatunnel_reporter` to `seatunnelReporter`